assets/syntaxes/Packages/C#/tests/syntax_test_PreprocessorDirectives.cs

```diff
@@ -12,7 +12,7 @@
 #pragma warning restore warning-list
 #pragma checksum "file.cs" "{3673e4ca-6098-4ec1-890f-8fceb2a794a2}" "{012345678AB}" // New checksum
 //       ^ keyword.other.preprocessor
-//                 ^ stirng.quoted.double
+//                 ^ string.quoted.double
```

Comment:
I'm never certain which typos are intentional / can't safely be fixed. I initially skipped this one, but I later decided it's probably fixable.